core/server/data/schema/schema.js

```diff
@@ -308,5 +308,41 @@ module.exports = {
         session_data: {type: 'string', maxlength: 2000, nullable: false},
         created_at: {type: 'dateTime', nullable: false},
         updated_at: {type: 'dateTime', nullable: true}
+    },
+    integrations: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        name: {type: 'string', maxlength: 191, nullable: false},
+        slug: {type: 'string', maxlength: 191, nullable: false, unique: true},
+        icon_image: {type: 'string', maxlength: 2000, nullable: true},
+        description: {type: 'string', maxlength: 2000, nullable: true},
+        created_at: {type: 'dateTime', nullable: false},
+        created_by: {type: 'string', maxlength: 24, nullable: false},
+        updated_at: {type: 'dateTime', nullable: true},
+        updated_by: {type: 'string', maxlength: 24, nullable: true}
+    },
+    api_keys: {
```

Comment:
How do you fetch the api keys usually?
I am wondering if e.g. integration_id needs an index.

Reply:
The most frequent fetch will be when an API key is used, in that case it will be fetched by it's id.

The only time we'll fetch by `integration_id` is when listing keys in the admin admin area.

Reply:
Okay 👍 Then it's probably fine as is.